.github/scripts/transform_dbt_test_results.py

```diff
@@ -94,10 +96,11 @@ def __init__(
         """
         The failing_rows list should be formatted like the rows
         returned by a csv.DictReader or a DictCursor, i.e. a list of
-        dicts mapping `{column_name: row_value}`.
+        dicts mapping `{field_name: row_value}`.
```

Comment:
This tweak is intended to avoid a misunderstanding now that `column_name` is a parameter of the function.